.travis.yml

```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```

Comment:
Why are you having to do these steps in CI? Is it related to [this](https://github.com/petester42/jest-preset-typescript-react-native/issues/1)? If so, updating to the latest `react-native` may help. I'd hate to see workarounds for something that's fixed (and if that doesn't fix it, it needs a comment about why this workaround is present).